irohad/network/impl/grpc_channel_builder.hpp

```diff
@@ -7,29 +7,67 @@
 #define IROHA_GRPC_CHANNEL_BUILDER_HPP
 
 #include <grpc++/grpc++.h>
+#include <boost/format.hpp>
+#include <limits>
```

Comment:
A minor issue.
Please move stdlib includes to the top of the file and put a blank line after.